nodebuilder/header/store/config.go

```diff
@@ -0,0 +1,25 @@
+package store
+
+import (
+	"fmt"
+
+	"github.com/celestiaorg/celestia-node/header/store"
+)
+
+// Config contains configuration parameters for the store
+type Config store.Parameters
```

Comment:
It should be part of the global header config, not separate. Similarly, we should pass only one config to the ConstructModuler